xbmc/video/VideoInfoScanner.cpp

```diff
@@ -1402,6 +1402,48 @@ namespace VIDEO
       if (!strTrailer.empty())
         movieDetails.m_strTrailer = strTrailer;
 
+      // ** Deal with 'Disc' subdirectories and add to/create a set
+      if (CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
+              CSettings::SETTING_MYVIDEOS_HANDLEDISCN))
+      {
+        size_t iDisc = pItem->GetDiscPath(movieDetails.m_strFileNameAndPath);
+        if (iDisc)
+        {
+          if (movieDetails.m_set.title.empty())
+          {
+            // Get existing sets
+            CFileItemList listItems;
+            std::string baseDir = "videodb://movies/sets/?ignoreSingleMovieSets=false";
```

Comment:
`const std::string...`
Is it correct to hard-wire to ignore single movie sets here? don't we have a setting for this?

Reply:
Yes. When the first disc is added it will create a set (if one doesn't exist already) - so when subsequent discs are added you want to find the single movie set with containing the first one.

Reply:
Okay, then. I was just asking. :-)